### PR TITLE
Move opentracing util to m3x

### PR DESCRIPTION
**Pull request by benraskin92**

### CODE REVIEW

File: opentracing/tracing_test.go [hunk 1]
```diff
@@ -18,7 +18,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package instrument
+package opentracingutil
```
Comment (prateek) [inline opentracing/tracing_test.go]: nit: rename so as to keep the package name the same as the folder name.

File: opentracing/context.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+package opentracingutil
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/log"
+	"golang.org/x/net/context"
```
Comment (prateek) [inline opentracing/context.go]: any reason to prefer this over the standard lib "context" here?

File: opentracing/context.go [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	} else {
+		tracer = getGlobalTracer()
+	}
+
+	span := tracer.StartSpan(operationName, opts...)
+
+	return span, opentracing.ContextWithSpan(ctx, span)
+}
+
+// Time is a log.Field for time.Time values. It translates to RF3339 formatted time strings.
```
Comment (prateek) [inline opentracing/context.go]: nit: `RFC3339`. Maybe also add an example of what that means, e.g. `"2019-04-01T15:04:05Z07:00"`

File: opentracing/context.go (hunk 2 above)
Comment (benraskin92) [reply opentracing/context.go]: Nope  - I'll switch.

**Pull request merged**